packages/legend-query-builder/src/components/QueryBuilder.tsx

```diff
@@ -47,6 +46,7 @@ import { QUERY_BUILDER_TEST_ID } from './QueryBuilder_TestID.js';
 import { flowResult } from 'mobx';
 import { QueryBuilderUnsupportedQueryEditor } from './QueryBuilderUnsupportedQueryEditor.js';
 import {
+  LEGEND_APPLICATION_DOCUMENTATION_KEY,
   BackdropContainer,
```

Comment:
why are we importing this?

Reply:
it's a lint fix for this pr https://github.com/finos/legend-studio/pull/1856

Reply:
please rebase as this was already fixed